projects/core/src/cart/store/effects/wish-list.effect.ts

```diff
@@ -45,9 +54,76 @@ export class WishListEffects {
     })
   );
 
+  @Effect()
+  loadWishList$: Observable<
+    | CartActions.LoadWisthListSuccess
+    | CartActions.CreateWishList
+    | CartActions.LoadCartFail
+  > = this.actions$.pipe(
+    ofType(CartActions.LOAD_WISH_LIST),
+    map((action: CartActions.LoadWisthList) => action.payload),
+    concatMap(userId => {
+      return this.cartConnector.loadAll(userId).pipe(
+        switchMap(carts => {
+          if (carts) {
+            const wishList = carts.find(cart => cart.name === 'wishlist');
+            if (Boolean(wishList)) {
+              return [
+                new CartActions.LoadWisthListSuccess({
+                  cart: wishList,
+                  userId,
+                }),
+              ];
+            } else {
+              return [
+                new CartActions.CreateWishList({ userId, name: 'wishlist' }),
+              ];
+            }
+          }
+        }),
+        catchError(error => from([new CartActions.LoadCartFail(error)]))
+      );
+    })
+  );
+
+  @Effect()
+  resetWishList$: Observable<
```

Comment:
context change should make the whole multi-carts entities reset, right? Now, it is only reset the wishlist.